storage/migrations/02_oasis_3_emerald.up.sql

```diff
@@ -4,32 +4,32 @@ BEGIN;
 
 CREATE TABLE oasis_3.emerald_rounds
 (
-  round     BIGINT PRIMARY KEY,
-  version   BIGINT NOT NULL,
+  round     UINT63 PRIMARY KEY,
+  version   UINT63 NOT NULL,
   timestamp TIMESTAMP WITH TIME ZONE NOT NULL,
 
-  block_hash      TEXT NOT NULL,
-  prev_block_hash TEXT NOT NULL,
+  block_hash      HEX64 NOT NULL, -- REFERENCES oasis_3.blocks(block_hash); not enforced so we can process emerald independently of consensus
```

Comment:
is this true? I thought this was its own hash of the runtime block header. it doesn't reference consensus

Reply:
Oh dang thanks, I didn't verify this one. Do we at all have a way then to say "round N was finalized in consensus block M"? Talking about the N->M mapping.

Comment removed.